### ISSUE TITLE
Color preference options for dark mode

### ISSUE THREAD

**Issue by ryanbugden** [typemytype/ramsayStreetRoboFontExtension#12]
Might be nice to implement dark mode color options, like the native RF prefs.

**Comment by typemytype** [comment]
yep! in RF4.4 you can use `if inDarkMode():` and pick a different color 

(and maybe switch to ezui for the settings panel)
